### DISCUSSION TITLE
Is it possible to use Meson for STM32L4?

### DISCUSSION THREAD

**Discussion by dreamer-coding** [mesonbuild/meson#13083] [Q&A]
Is it possible to use Meson for STM32L4? If so how can I go about making an executable project for an STM32 board? lastly, how would I do this if I wanted to provide Meson build support for libraries I use for my STM32 projects?

**Comment by jeandet** [comment]
Hello @dreamer-coding-555, you can build for any cortex-m target as long as you have a supported compiler and provide your cross files.
[This](https://github.com/hwengineer/STM32F3Discovery-meson-example) repo looks like a good starter. 
[Here](https://github.com/LaboratoryOfPlasmaPhysics/LFR_Flight_Software) is another cross compilation setup that I know works but for Sparc.
